modules/okhttp-client/src/main/java/com/spotify/apollo/http/client/HttpClientProvider.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * -\-\-
+ * Spotify Apollo okhttp Client Module
+ * --
+ * Copyright (C) 2013 - 2015 Spotify AB
+ * --
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * -/-/-
+ */
+package com.spotify.apollo.http.client;
+
+import com.google.inject.Inject;
+import com.google.inject.Provider;
+import com.squareup.okhttp.OkHttpClient;
+
+class HttpClientProvider implements Provider<HttpClient> {
+
+  private final OkHttpClient okHttpClient;
+
+  @Inject
+  HttpClientProvider(OkHttpClient okHttpClient) {
+    this.okHttpClient = okHttpClient;
+  }
+
+  @Override
+  public HttpClient get() {
+    return new HttpClient(okHttpClient);
```

Comment:
This constructor already has an `@Inject` annotation, so it should suffice to just bind the class in the module, as `bind(HttpClient.class)`.